guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md

```diff
@@ -3,8 +3,32 @@ title: Use @if and @else to Add Logic To Your Styles
 ---
 ## Use @if and @else to Add Logic To Your Styles
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+## Solution
```

Comment:
Your given solution resolves to this:
![image](https://user-images.githubusercontent.com/34807532/54029722-6a5c9980-41cf-11e9-8558-063829feb7ae.png)

Reply:
@Bouncey / @ValeraS Is this a problem on our end?

Reply:
@thecodingaviator I copied/pasted the solution provided in this PR and used it in current production and beta site and it passed all tests.  I also tested locally using current master and it passes with no errors.  How were you testing?

Reply:
I was testing on latest beta, seems fixed now, may have been a problem on my end